### DISCUSSION TITLE
How can I build/run this in ARM?

### DISCUSSION THREAD

**Discussion by nerdmor** [advplyr/audiobookshelf#1593] [Q&A] [answered]
I have a Raspberry Pi 3 sitting here that I'd love to use as a server. But, as it is pretty weak, it would be better to run it directly on the OS, not inside Docker. However, the .deb install packages are only for x86.

Are there any guides on running this directly on the OS, or on how to make my own ARM build?

Thanks in advance

**Comment by advplyr** [comment, marked as answer]
If you don't want to use Docker then you'll have to build from source. There are instructions for that here https://github.com/advplyr/audiobookshelf/discussions/259#discussioncomment-1869729
You don't need the optional part

**Reply by nerdmor** [reply]
Thanks!
As a suggestion: it could be useful to link that, or make an official version of it, in the readme or the site :)
